### PR TITLE
Update project-lifecycle.md

### PR DESCRIPTION
**Pull request by SeanBohan**
Changed:

1. project Proposal Requirements to move the link to project intake form down the document and add context for its use (the intake form is intended to be filled out AFTER a project has been approved by the TAC).

2. Labs Application: move Project Intake form below project proposal, make clear it is to be filled out AFTER acceptance

### CODE REVIEW

File: docs/governance/project-lifecycle.md [hunk 1]
```diff
@@ -99,13 +100,14 @@ End users should evaluate Labs with care, as this stage does not set requirement
...
 * Submit a [project proposal](https://github.com/openwallet-foundation/project-proposals#instructions-for-proposing-a-project).
-
+* If the Lab is accepted by the TAC, the OWF Lab Project will then complete [project intake form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform?fbzx=4351560609072672295). This intake form will create:
+  * A project charter [(sample)](https://docs.google.com/document/d/1Sxnktu3d_xLFSZmAmSFbUp1Y1s1E36YvNDupY2AmigU/edit) that documents an intellectual property policy that leverages the Apache 2.0 license or a permissive open source license.
+  * A collaboration agreement that in the case of existing projects, provides an agreement to transfer the project name, trademarks, and electronic account assets (github repo, social media accounts, domain names, etc.) to Linux Foundation Europe for the benefit of the OpenWallet Foundation.
+
```
Comment (tkuhrt) [inline docs/governance/project-lifecycle.md:107]:
```suggestion
```

I don't think we need this since you added this at line 47.

**Review by tkuhrt — CHANGES_REQUESTED** [review]

File: docs/governance/project-lifecycle.md (hunk 1 above)
Comment (skounis) [reply docs/governance/project-lifecycle.md:107]: The 3rd element about the collaboration agreement is not part of what we have in line `47`. Should we keep it here?

Comment (tkuhrt) [reply docs/governance/project-lifecycle.md:107]: We should probably move it to line 47 as it is relevant for all project stages.

Comment (tkuhrt) [reply docs/governance/project-lifecycle.md]: I have made this change.

**Review by tkuhrt — APPROVED** [review]

File: docs/governance/project-lifecycle.md [hunk 2]
```diff
@@ -44,6 +43,7 @@ Projects must be formally [proposed via GitHub](https://github.com/openwallet-fo
 
 * [Impact stage](#impact-stage) and [Growth stage](#growth-stage) projects are accepted via a two-thirds supermajority vote of the TAC. [Labs](#labs) are accepted via a simple majority of the TAC.
 * Satisfaction of the requirements of the initial stage of the project. The TAC will determine the appropriate initial stage for the project. The project can apply for a different stage via the review process.
+* If the project is accepted by the TAC, a project charter and contribution agreement (if necessary) can be obtained by completing [this form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform?fbzx=4351560609072672295). Here is [sample project charter](https://docs.google.com/document/d/1Sxnktu3d_xLFSZmAmSFbUp1Y1s1E36YvNDupY2AmigU/edit) that you can review).  A collaboration agreement is needed for existing projects and provides an agreement to transfer the project name, trademarks, and electronic account assets (github repo, social media accounts, domain names, etc.) to Linux Foundation Europe for the benefit of the OpenWallet Foundation.
```
Comment (tkuhrt) [inline docs/governance/project-lifecycle.md:46]:
```suggestion
* If the project is accepted by the TAC, a project charter and contribution agreement (if necessary) can be obtained by completing [this form](https://docs.google.com/forms/d/e/1FAIpQLSeO1bDGHUP-ZpCo1uynm94YOxZlek6RhCH7o3FnX1lZSXXfSQ/viewform?fbzx=4351560609072672295). Here is [sample project charter](https://docs.google.com/document/d/1Sxnktu3d_xLFSZmAmSFbUp1Y1s1E36YvNDupY2AmigU/edit) that you can review.  A collaboration agreement is needed for existing projects and provides an agreement to transfer the project name, trademarks, and electronic account assets (github repo, social media accounts, domain names, etc.) to Linux Foundation Europe for the benefit of the OpenWallet Foundation.
```

**Review by rolsonquadras — APPROVED** [review]

**Pull request merged**